docs/csharp/programming-guide/nullable-types/handling-null-and-operators.md

```diff
@@ -0,0 +1,159 @@
+	{
+		get 
+		{
+			 return _myStringValue ?? "Default value";
+		}
+	}
+}
+```
+
+Basically what is happening here is that we return the backing field value, unless it is null then return the static string.
+
+#### Remarks
+
+### Null operator (?.)
+The null operator, or null conditional can be used to reduce the number of if-statements and nesting in your code. It works slightly different from the null coalescing operator.
+
+With this operator you can incorporate your null check with another condition. Please have a look at the samples below for more clarification. 
+
+#### Sample
+
+```
+public class NullOperatorSample1
+{	
+	public bool HasLengthOfFive(string value)
+	{
+		if (value == null)
+			return false;
+			
+		return value.Length == 5;
+	}
+}
+```
+
+We can reduce this code to just one line with the null conditional operator. The changed code is shown underneath.
+
+```
+public class NullOperatorSample2
+{	
+	public bool HasLengthOfFive(string value)
+	{
+		return value?.Length == 5;
+	}
+}
+```
+
+When using the above method, invoked with a `null` value, it will simply return `false`. Because of the null operator on the `value` variable, it will be evaluated for containing a `null` value. If it actually contains `null`, it knows that the rest of the condition will be false.
+
+It can also be used to invoke methods. If the method returns a value and the object which is called upon is `null`, a `null` value will be returned as a result of that method as well. This is known as null-propagation. Sample code can be seen below.
+
+```
+public class NullOperatorSample3
+{	
+	public string GiveMeFive(string value)
+	{
+		return value?.Substring(0, 5);
+	}
+}
+
+```
+
+Invoking the above method like this: `GiveMeFive(null)`, will return `null` itself.
+
+Using the null operator on a method that does not return any value, will just not be invoked. A typical example of this is when working with event handlers. Observe a code sample of this below.
+
+```
+public class NullOperatorSample4
+{
+	public event EventHandler<string> OnStringValueChanged;
+	
+	private int _stringValue;
+	
+	public int StringValue
+	{
+		get
+    	{
+			return _stringValue;
+		}
+		
+		set
+		{
+			if (OnStringValueChanged != null)
+			{
+				_stringValue = value;
+				OnStringValueChanged(this, value);
+			}
+		}
+	}
+}
+```
+
+The above code shows some code that is typically used to check if there are any subscribers to the `OnStringValueChanged` event and invoke the event whenever there are subscribers. All the code in the setter of this property can now be reduced to a single line, like this: `OnStringValueChanged?.Invoke(this, value);`. The event will now only be invoked whenever `OnStringValueChanged` is not `null`.
+
+#### Remarks
+The null operator cannot be used with non-nullable, value types. For instance, the next code will produce a compile-time error.
+
+`int length = text?.Length; // Compile Error: Cannot implicitly convert type 'int?' to 'int'`
```

Comment:
This code block, since it demonstrates a compiler error, does not go in the buildable sample I reference above.